challenge_storing_groceries/src/challenge_storing_groceries/storing_groceries.py

```diff
@@ -74,28 +74,40 @@ def move_table(userdata=None, manipulate_machine=None):
                 manipulate_machine.place_designator._area       = closest_workspace.place_entity_conf.manipulation_volumes[0]
                 manipulate_machine.place_designator.place_location_designator.id = closest_workspace.place_entity_conf.entity_id
                 manipulate_machine.cabinet.id_                  = closest_workspace.place_entity_conf.entity_id
+                open_door.cabinet.id_                           = closest_workspace.place_entity_conf.entity_id
```

Comment:
OpenDoorMachine doesn't have cabinit_id_ as a member. Who does this work? Maybe better to change implementation.

Reply:
What do you mean here? It could be that the splitting-off of the door opening wasn't perfect and some extra cruft from the storing groceries evolution was still included

Reply:
Don’t mind. Misread it